vllm/v1/engine/core_client.py

```diff
@@ -169,6 +170,26 @@ def add_lora(self, lora_request: LoRARequest) -> None:
         self.engine_core.add_lora(lora_request)
 
 
+@dataclass
+class ShutdownResources:
```

Comment:
nit: Just try to make it more clear. Feel free to ignore.

```suggestion
class BackgroundResources:
    """A data class of background resources (e.g., context, process handle, sockets).
    This is intended to be registered a a finalizer of the engine core client, so that when
    the engine core client is shutdown, the __call__ in invoked and all resources will
    be cleaned up.
    """
```